core/src/main/java/org/apache/struts2/views/jsp/ui/AbstractUITag.java

```diff
@@ -90,7 +93,11 @@ protected void populateParams() {
         uiBean.setDisabled(disabled);
         uiBean.setLabel(label);
         uiBean.setLabelSeparator(labelSeparator);
-        uiBean.setLabelPosition(labelPosition);
+        if (labelposition != null) {
```

Comment:
Not sure but since labelposition is deprecated, shouldn't we check for `labelPosition` here instead? to give it preference and priority higher than the deprecated one.

Reply:
Hm... right :)